ksp-processors/build.gradle.kts

```diff
@@ -11,3 +13,11 @@ dependencies {
 
     implementation(libs.kotlinx.serialization.json) // use types directly
 }
+
+tasks {
+    withType<KotlinCompile> {
+        compilerOptions {
+            freeCompilerArgs.add("-Xcontext-receivers")
```

Comment:
just a note: this is only ok since it's an internal project that isn't published

since context receivers are highly experimental at this point, publishing code using this feature wouldn't be a good idea

Reply:
That's exactly what I thought, never intended to add it to public code